addons/website_sale_wishlist/static/src/js/website_sale_wishlist.js

```diff
@@ -20,30 +20,58 @@ sAnimations.registry.ProductWishlist = sAnimations.Class.extend(ProductConfigura
     },
     events: sAnimations.Class.events,
```

Comment:
(not you, but... lol)

Reply:
There's a comment above, it's to disable the events from `ProductConfiguratorMixin`... But it would be better to not extend it at all and to make `selectOrCreateProduct` a util I suppose.

Reply:
Did not really cared about that here. Just that I suppose it was supposed to be `sAnimations.Class.**prototype**.events`, otherwise it is just a fancy way to write `undefined` :laughing: